pymc3/tests/test_distributions_random.py

```diff
@@ -1451,6 +1451,21 @@ class TestOrderedProbit(BaseTestDistribution):
     ]
 
 
+class TestOrderedMultinomial(BaseTestDistribution):
+    pymc_dist = pm.OrderedMultinomial
```

Comment:
```suggestion
    pymc_dist = pm._OrderedMultinomial
```

Or an explicitly imported `_OrderedMultinomial` if it's not available at the module level. Same thing for the other ones

Reply:
Done